Libraries/LibGfx/PaintStyle.h

```diff
@@ -38,7 +38,7 @@ class PaintStyle : public RefCounted<PaintStyle> {
     virtual Color sample_color(IntPoint) const { return Color(); }
 };
 
-class SolidColorPaintStyle final : public PaintStyle {
```

Comment:
could you add some information in the commit description explaining why removing `final` fixes undefined behavior?